src/main/scala/api/MyMap.scala

```diff
@@ -0,0 +1,98 @@
+package api
+
+
+import scala.collection.JavaConverters._
+
+import java.util.concurrent.ConcurrentHashMap
+
+import javax.servlet.http.{HttpServlet, HttpServletRequest, HttpServletResponse}
+
+import scala.collection.concurrent
+
+object MyMap {
```

Comment:
Can this be a `class` (not object) and instantiated when needed? So we'll create a new instance for usage in tests as well, not just use the same object everywhere.